library/ui-strings/src/main/res/values/strings.xml

```diff
@@ -3330,6 +3331,9 @@
     <string name="device_manager_session_rename_edit_hint">Session name</string>
     <string name="device_manager_session_rename_description">Custom session names can help you recognize your devices more easily.</string>
     <string name="device_manager_session_rename_warning">Please be aware that session names are also visible to people you communicate with.</string>
+    <string name="device_manager_sessions_sign_in_with_qr_code_title">SIGN IN WITH QR CODE</string>
```

Comment:
I guess strings should be lower case. The upper case style should be done using text appearance instead.

Reply:
Done. Actually it is more consistent with lowercase.